plugins/bufbuild/connect-kotlin/v0.1.3/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1.4
+FROM gradle:7.4.0-jdk17 AS build
```

Comment:
There is gradle:7.6.1 or gradle:8.0.2 .. but I'm not sure if this is pinned for a specific reason.

Reply:
@buildbreaker - Do you think we could bump to latest Gradle 7.x?